raiden/network/proxies/secret_registry.py

```diff
@@ -62,14 +63,7 @@ def __init__(
         self.open_secret_transactions = dict()
 
     def register_secret(self, secret: typing.Secret):
-        secrethash = sha3(secret)
-        if self.check_registered(secrethash):
-            log.info(
-                'secret already registered',
-                node=pex(self.node_address),
-                contract=pex(self.address),
-                secrethash=encode_hex(secrethash),
-            )
+        if self._check_and_log_registered(secret):
```

Comment:
I think it would be cleared if `register_secret` just forwarded the cal to `register_secret_batch`